boards/matrix_portal_m4/README.md

```diff
@@ -0,0 +1,26 @@
+# Adafruit Matrix Portal M4 Board Support Crate
+
+This crate provides a type-safe API for working with the [Adafruit Matrix Portal M4
+board](https://www.adafruit.com/product/4745).
+
+## Prerequisites
+* Install the cross compile toolchain `rustup target add thumbv6m-none-eabi`
```

Comment:
Change the toolchain to the correct one: `thumbv7em-none-eabihf`

Reply:
Changed thumbv6m-none-eabi to thumbv7em-none-eabihf